File: Level 106/homework/HW1/hw1.js
function isLeapYear(year) {
    if (year % 400 === 0) return true;
    if (year % 100 === 0) return false;
    return year % 4 === 0;
}

console.log("Year 2000:", isLeapYear(2000));
console.log("Year 2100:", isLeapYear(2100));
console.log("Year 2024:", isLeapYear(2024));
console.log("Year 2023:", isLeapYear(2023));

File: Level 106/homework/HW1/hw1.py
def is_leap_year(year):
    if year % 400 == 0:
        return True
    if year % 100 == 0:
        return False
    return year % 4 == 0

print("Year 2000:", is_leap_year(2000))
print("Year 2100:", is_leap_year(2100))
print("Year 2024:", is_leap_year(2024))
print("Year 2023:", is_leap_year(2023))
